src/MgvOfferTaking.sol

```diff
@@ -89,6 +112,11 @@ abstract contract MgvOfferTaking is MgvHasOffers {
       sor.olKey = olKey;
       OfferList storage offerList;
       (sor.global, sor.local, offerList) = _config(olKey);
+      mor.maxRecursionDepth = maxRecursionDepth > 0 ? maxRecursionDepth : sor.global.maxRecursionDepth();
+      /* We have an upper limit on total gasreq for failing offers to avoid failing offers delivering nothing and exhausting gaslimit for the transaction. */
+      mor.maxGasreqForFailingOffers =
```

Comment:
I'm missing why we have to copy `maxGasreqForFailingOffers` from `sor` to `mor`

Reply:
I don't understand? Maybe the code changed?

Reply:
ah, it is because I allow it to be overridden by parameter to function.

Reply:
Ah yes makes sense :)